ethereum-types/src/hash.rs

```diff
@@ -121,16 +121,17 @@ mod tests {
 	use super::{H160, H256};
 	use serde_json as ser;
 
+	// #[cfg(feature = "fixed-hash/byteorder-support")]
 	#[test]
 	fn test_serialize_h160() {
```

Comment:
This test (and the one after) is kind of weird, it's testing hash features. I think we should remove them or move them to `fixed-hash` (unless there are already tests for this there, and I think there are).

Reply:
Please do note that I wanted the changes in this crate to be minimal to the outside ecosystem so that I can refocus on other work as soon as possible instead of fixing all the to-be-broken depending crates. That's the reason I did not change some of the things that should have been done otherwise such as renaming the crate features etc.

Reply:
yeah we should do this! you are right